packages/grid_client/tests/modules/gateways.test.ts

```diff
@@ -190,15 +192,34 @@ test("TC1237 - Gateways: Expose a VM Over Gateway", async () => {
   expect(gatewayResult[0].backends).toStrictEqual(backends);
 
   const domain = "https://" + gatewayResult[0].domain;
+  let reachable = false;
+
+  for (let i = 0; i < 15; i++) {
+    fetch(domain, { mode: "no-cors" })
+      .then(r => {
+        log("gateway is reachable");
+        reachable = true;
+        i = 15;
+      })
+      .catch(e => {
+        log("gateway is not reachable");
+      });
+    const wait = await setTimeout(10000, "Waiting for gateway to be ready");
```

Comment:
I think 10 sec is too much to wait for each iteration

Reply:
sure will decrease it to 5 seconds and increase the iterations to 30

Reply:
done in https://github.com/threefoldtech/tfgrid-sdk-ts/pull/107/commits/de767e12e94a87712fd099f88828ca329928ff9d Now it's 30 iterations for 5 sec each.